index.js

```diff
@@ -28,7 +28,13 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let movieTitle = [];
+  for (let i = 0; i < movies.length; i++){
```

Comment:
All I can say here is watch the spacing at the end of the loop declaration `) {`